solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -536,7 +517,7 @@ pub struct ManageEntity<'info> {
 /// Confirm that the user authority matches signer authority field
 #[derive(Accounts)]
 #[instruction(base: Pubkey, user_handle: UserHandle)]
-pub struct TrackSocialAction<'info> {
+pub struct SocialAction<'info> {
```

Comment:
The convention in the rest of the file for account struct containers is to mimic the method it's used in ie WriteSocialAction